osu.Game/Screens/Play/SaveFailedScoreButton.cs

```diff
@@ -87,5 +90,21 @@ private void load(OsuGame? game, Player? player, RealmAccess realm)
                 }
             }, true);
         }
+
+        public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
+        {
+            switch (e.Action)
+            {
+                case GlobalAction.SaveReplay:
+                    button.TriggerClick();
```

Comment:
Can be done similarly to results screen.

Reply:
How so? To watch the replay? If so then the hotkey needs to be renamed.

Reply:
With the same behavior. Once scores have replays attached in global leaderboards, you can also save and then watch.